File: template/app/src/main/java/com/xmartlabs/template/model/AuthResponse.java
package com.xmartlabs.template.model;


import org.parceler.Parcel;

import lombok.Data;

@Data
@Parcel
public class AuthResponse {
  //TODO: check service auth response to match app's AuthResponse fields
  String accessToken;
  String scope;
  String tokenType;
}

File: template/app/src/main/java/com/xmartlabs/template/model/AuthResponse.kt
package com.xmartlabs.template.model

import org.parceler.Parcel

//TODO: check service auth response to match app's AuthResponse fields
@Parcel(Parcel.Serialization.BEAN)
class AuthResponse {
  var accessToken: String? = null
  var scope: String? = null
  var tokenType: String? = null
}
